packages/sanity/src/tasks/src/tasks/components/list/TasksList.tsx

```diff
@@ -37,88 +27,94 @@ const Details = styled.details`
     transform: rotate(0);
   }
 `
-const Summary = styled.summary`
+const SummaryBox = styled(Box)`
```

Comment:
As curiosity, what is the reason behind this change?

Reply:
No need to revert it, it adds future proofing which is a good thing now that you have it. 
I was curious if maybe some other component was handling boxes in an special way. 
Thanks!